src/modules/mavlink/mavlink_main.cpp

```diff
@@ -988,6 +990,7 @@ Mavlink::send_bytes(const uint8_t *buf, unsigned packet_len)
 		return;
 	}
 
+
```

Comment:
can you please avoid adding unnecessary newlines?

Reply:
Sure, removed.